ambari/assets/dashboards/base_dashboard.json

```diff
@@ -0,0 +1,285 @@
+{
+  "board_title": "Ambari base dashboard",
+  "read_only": false,
+  "author_info": {
+    "author_name": "Julia Simon"
+  },
+  "description": "## Ambari Dashboard\n\nThis is an example Ambari dashboard demonstrating the metrics that the integration collects.",
+  "board_bgtype": "board_graph",
+  "created": "2019-06-13T14:39:10.350422+00:00",
+  "created_by": {
```

Comment:
You probably want to remove your personal infos from here, with something like https://github.com/DataDog/integrations-core/pull/3900/files